package.json

```diff
@@ -24,19 +25,12 @@
     "test": "react-scripts test",
     "test:ci": "npm run test -- --coverage --watchAll=false",
     "eject": "react-scripts eject",
+    "lint:fix": "eslint --fix --ext .js,.jsx,.tsx,.ts .",
```

Comment:
I wonder if this is doing the same thing as `format`. Did you see different results?

Reply:
I was trying to find a way to resolve all fixable problems with one command instead of having to fix each lint warning in each file (for example, removing unused imports). This actually did not work as I was hoping it would and I forgot to remove it. I will delete it.

Reply:
That was the intended purpose of the format script as well. Did that one not fix some of them?

Reply:
it looks like it does everything except removing unused imports.

Reply:
Which is not too big a deal